policy/modules/contrib/networkmanager.te

```diff
@@ -40,6 +40,10 @@ init_daemon_domain(NetworkManagerDispatcher_t, NetworkManagerDispatcher_exec_t)
 type NetworkManagerDispatcher_initrc_exec_t;
 init_script_file(NetworkManagerDispatcher_initrc_exec_t)
 
+type NetworkManagerSudo_t;
+type NetworkManagerSudo_exec_t;
+init_daemon_domain(NetworkManagerSudo_t, NetworkManagerSudo_exec_t)
```

Comment:
The interface description reads "Create a domain for long running processes (daemons/services)". There seems to be no daemon running, have you tried e. g. `init_system_domain()`?

Reply:
Spotted right now: `dbus_system_domain()` calls `init_system_domain()`.

Reply:
Yes, so will be used `dbus_system_domain()`.